src/material/stepper/stepper-intl.ts

```diff
@@ -21,6 +21,9 @@ export class MatStepperIntl {
 
   /** Label that is rendered below optional steps. */
   optionalLabel: string = 'Optional';
+
+  /** Label that is used to indicate step as completed to screen readers. */
+  completedScreenReaderLabel: string = 'Completed';
```

Comment:
This should just be `completedLabel`

(that fact that it's read by a screen-reader related to how it will be _used_ rather than what it _is_)